### PR TITLE
[Event Hubs] Update troubleshooting log asks

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to update the set of logs that we ask for collection, adding in some recent diagnostic events.

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -356,6 +356,8 @@ When filing GitHub issues, the following details are requested for all scenarios
   - 88 (Buffered Producer Publish Start)
   - 89 (Buffered Producer Publish Complete)
   - 90 (Buffered Producer Publish Error)
+  - 124 (Buffered Producer Idle Start)
+  - 125 (Buffered PRoducer Idle Complete)
```
Comment (christothes) [inline sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md:360]:
```suggestion
  - 125 (Buffered Producer Idle Complete)
```

File: sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md [hunk 2]
```diff
@@ -383,6 +385,9 @@ When filing GitHub issues, the following details are requested for all scenarios
   - 45 (EventProcessorClaimOwnershipError)
   - 103 (EventProcessorLoadBalancingCycleSlowWarning)
   - 104 (EventProcessorHighPartitionOwnershipWarning)
+  - 123 (EventProcessorProcessingHandlerStart)
```
Comment (christothes) [inline sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md:388]: Just checking that this one is intentionally omitted from the new events in the above change.

File: sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md (hunk 1 above)
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md:360]: Wow... I should really avoid throwing out a PR at the end of the day!

Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md:360]:
```suggestion
  - 126 (Buffered Producer Idle Start)
  - 127 (Buffered Producer Idle Complete)
```

File: sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md (hunk 2 above)
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md:388]: I totally messed up the numbering in the other section...

**Review by christothes — APPROVED** [review]

**Pull request merged**